### DISCUSSION TITLE
Java support

### DISCUSSION THREAD

**Discussion by itsthavid** [zed-industries/zed#6647] [Feature Requests]
I really love Zed and how efficiently it handles tasks for my everyday work. The only problem I have encountered for now is the lack of Java support. I use it daily and not being able to develop in this language on Zed I am very sorry. Are you going to add it among the languages supported soon?

**Comment by ghost** [comment]
There's an open issue for adding Java support (https://github.com/zed-industries/zed/issues/5301). Now that Zed is open source, I'm sure Java will be added eventually. I'm personally looking into added support for languages I'd like to see in Zed (e.g. https://github.com/zed-industries/zed/issues/5281#issuecomment-1909092803, which you might be able to reference to add basic syntax highlighting).

**Reply by pedrosmmc** [reply]
For how long do we need to wait? 😒

**Reply by BHASVIC-sholtocoulter23** [reply]
same bro ^
